.github/workflows/php-test.yml

```diff
@@ -50,10 +50,59 @@ jobs:
         fail_ci_if_error: true
         verbose: true
 
+  integration-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: [ '8.0', '8.1', '8.2' ]
+        nextcloud-versions: [ 'master', 'stable27', 'stable26' ]
+        include:
+          - php-versions: '7.4'
+            nextcloud-versions: 'stable25'
+          - php-versions: '8.1'
+            nextcloud-versions: 'stable25'
```

Comment:
simplify so it only tests each php version and nextcloud version at least once but not all possible combinations

this will reduce CI time